src/plugin.rs

```diff
@@ -818,6 +818,39 @@ impl Plugin {
         Ok(0)
     }
 
+    #[native(name = "JSON_RemoveNode")]
+    pub fn json_remove_node(&mut self, _: &Amx, node: i32, input: i32) -> AmxResult<i32> {
+        let src: serde_json::Value = match self.json_nodes.get(input) {
+            Some(src) => src.clone(),
+            None => return Ok(1)
+        };
+        let v: &mut serde_json::Value = match self.json_nodes.get(node) {
+            Some(v) => v,
+            None => return Ok(2),
+        };
+        if !v.is_array() {
+            return Ok(3);
+        }
+        v.as_array_mut().unwrap().retain(|val| *val != src);
```

Comment:
curious how this works if the node is an object or an array, does it perform a deep equality check?

Reply:
I don't have a knowledge in rust but what I can confirm it's the same behavior as `JSON_ArrayRemove`